pass/pass_linux.go

```diff
@@ -46,26 +46,11 @@ func (p Pass) checkInitialized() error {
 	if passInitialized {
 		return nil
 	}
-	// In principle, we could just run `pass init`. However, pass has a bug
-	// where if gpg fails, it doesn't always exit 1. Additionally, pass
-	// uses gpg2, but gpg is the default, which may be confusing. So let's
-	// just explictily check that pass actually can store and retreive a
-	// password.
-	password := "pass is initialized"
-	name := path.Join(getPassDir(), "docker-pass-initialized-check")
-
-	_, err := p.runPassHelper(password, "insert", "-f", "-m", name)
+	// We just run a `pass ls`, if it fails then pass is not initialized.
+	_, err := p.runPassHelper("", "ls")
 	if err != nil {
 		return fmt.Errorf("error initializing pass: %v", err)
```

Comment:
Hello @eyJhb,

What do you think about changing the error message to something on these lines?

`password-store not initialized: %v`. 

Since we are not trying to intialize the password-store, this message could confuse users.